src/ports/postgres/modules/graph/test/apsp.sql_in

```diff
@@ -97,3 +97,14 @@ SELECT assert(weight = 2, 'Wrong output in graph (APSP)')
 SELECT assert(parent = 2, 'Wrong output in graph (APSP)')
 	FROM out_gr WHERE src = 0 AND "DEST" = 5 AND grp = 1;
 
+DROP TABLE IF EXISTS out, out_summary;
```

Comment:
Can you add a brief description of the purpose of this test so that it's easier to navigate through the test file ? same goes for all other tests